src/Inputs/Slider.tsx

```diff
@@ -1,79 +1,384 @@
+                                    left: `${translateToPixels(key)}px`,
+                                }}
+                            >
+                                {mark}
+                            </div>
+                        ),
+                    )}
+                </SliderBoardMarks>
+            )}
+        </LabelLayout>
+    );
+};
+
+const SliderBoard = styled.div<SliderProps>`
         width: 100%;
-        height: 8.4px;
-        cursor: pointer;
+        height: 4px;
         animate: 0.2s;
-        box-shadow: 1px 1px 1px #000000, 0px 0px 1px #0d0d0d;
-        background: #3071a9;
-        border-radius: 1.3px;
-        border: 0.2px solid #010101;
+        background: #e9e9e9;
+        cursor:pointer;
+        ${({ theme }): string => `
+            border-radius: ${theme.dimensions.radius};
+            box-shadow: ${theme.depth[1]};
+        `}
+        // Disabled
+        ${({ disabled }): string =>
+            disabled
+                ? `
+            cursor: not-allowed;
+            opacity: 0.6;
+        `
+                : ''}
+        `;
+
+export interface SelectedBarProps {
+    left: number;
+    right: number;
+    disabled: boolean;
+    hasRail: boolean;
+}
+
+const SliderBoardSelected = styled.div<SelectedBarProps>`
+    top: 1px;
+    height: 4px;
+    animate: 0.2s;
+    position: relative;
+    left: ${({ left }): string => `${left}px` || '0px'};
+    ${({ theme, hasRail }): string => `
+            border-radius: ${theme.dimensions.radius};
+            box-shadow: ${theme.depth[1]};
+            background: ${hasRail ? theme.colors.primary : '#e9e9e9'};
+        `}
+    width: ${({ right }): string => `${right}px` || '0px'};
+    ${({ disabled }): string =>
+        disabled
+            ? `
+        cursor: not-allowed;
+        opacity: 0.6;
+    `
+            : ''}
+`;
+
+const SliderBoardMarks = styled.div<SliderProps>`
+    display: flex;
+    width: 100%;
+    animate: 0.2s;
+    position: relative;
+    font-weight: bold;
+    top: 6px;
+    ${({ disabled }): string =>
+        disabled
+            ? `
+            cursor: not-allowed;
+            opacity: 0.6;
+        `
+            : ''}
+    ${({ theme }): string => `
+                border-radius: ${theme.dimensions.radius};
+                box-shadow: ${theme.depth[1]};
+            `}
+`;
+
+export interface KnobProps {
+    left: number;
+    hasTwoKnobs: boolean;
+    disabled: boolean;
+}
+
+const SliderKnobRight = styled.div<KnobProps>`
+    width: 14px;
+    height: 14px;
+    border-radius: 50%;
+    position: relative;
+    left: ${({ left }): string => `${left}px` || '100px'};
+    ${({ hasTwoKnobs }): string =>
+        hasTwoKnobs
+            ? `
+         top: -21px;
+`
+            : 'top: -7px; '}
+
+    cursor: grab;
+    ${({ theme }): string => `
+            background: ${theme.colors.primary};
+        `}
+    &:active {
+        border: solid 2px #96dbfa;
```

Comment:
add `box-sizing: border-box` so the border doesn't change the position of the knob